resources/schema.gql

```diff
@@ -298,6 +298,10 @@ type AddData {
 	when resuming.
 	"""
 	newSourceState: SourceState
+	"""
+	ODF extensions.
+	"""
+	extra: ExtraAttributes
```

Comment:
UI might need a higher-level view model for these attributes. Of course, there could be really custom key-value system shows as is, but perhaps for ODF-defined and Kamu-defined attributes, UI needs a representation of higher-level intent.

Reply:
I started with exposing `DatasetArchetype` in GQL as a separate endpoint so UI didn't have to extract it from schema annotations.

ODF Data schema itself only can be requested as encoded in JSON/YAML, so there are no GQL objects which we could extend with things like field `description`. UI will need to learn how to work with ODF schema and its extensions, which I think is OK.

The extra attrs you pointed out are attached to a `AddData` metadata event which currently can only have `opendatafabric.org/inkedObjects` attribute. We could surface it into an endpoint ... but I'm actually considering a path similar to `DataSchema` - perhaps we should not create GQL objects for every metadata block/event, and only expose those as encoded in JSON/YAML/Flatbuffers in our APIs.